src/Deserializers/LegacySiteLinkListDeserializer.php

```diff
@@ -65,6 +65,11 @@ private function assertElementIsValidArray( array $arrayElement ) {
 		}
 	}
 
+	/**
+	 * @param array $siteLinkArray
+	 *
+	 * @return SiteLinkList
+	 */
 	private function getDeserialized( array $siteLinkArray ) {
```

Comment:
This comment does not add info that is not already clear from the signature and the return statement, or cannot be understood by static code analysis tools without it.